src/mscorlib/src/System/Globalization/CalendarData.Windows.cs

```diff
@@ -123,13 +129,17 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
         {
             EnumCalendarsData data = new EnumCalendarsData();
             data.userOverride = 0;
+#if INSIDE_CLR
```

Comment:
I know you have not introduced `INSIDE_CLR`. We have several different ifdefs that are used for this purpose because of they were introduced by different people at different times. I am suggesting that `#if CORECLR` is the one we should standardize on because of it is the most logical one.